roles/openshift-applier/tasks/process-one-entry.yml

```diff
@@ -88,9 +88,8 @@
     oc {{ file_action }} {{ target_namespace }} -f {{ file }}
   register: command_result
   failed_when:
-  - command_result.rc != 0
-  - "'AlreadyExists' not in command_result.stderr"
-  - "'NotFound' not in command_result.stderr"
+    (command_result.rc != 0 and 'AlreadyExists' not in command_result.stderr and 'NotFound' not in command_result.stderr) or
```

Comment:
To revert to the functionality that used to be here, and to keep it a bit simpler, couldn't it just be made:
```
    (command_result.rc != 0 and 'AlreadyExists' not in command_result.stderr ) or
    (file_action == 'delete' and command_result.rc != 0 and 'AlreadyExists' not in command_result.stderr and 'NotFound' not in command_result.stderr)
```

... same for the one below, just use `template_action` instead of `file_action`

Reply:
nvm - that will most likely cause issues with the delete :-) *sigh* I just hate to see all the negative logic ...